.github/workflows/lint-and-test-lambdas.yaml

```diff
@@ -170,3 +170,27 @@ jobs:
 
       - name: Test
         run: npm test
+
+  lint_and_test_write_activity_log:
+    name: write_activity_log
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./lambda/write-activity-log
+    steps:
+      - uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # pin@v3
+
+      - name: Install Node.js
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # pin@v3
+        with:
+          node-version-file: ".nvmrc18"
```

Comment:
Note to self, we should really get the rest of these ones on node 18 and then turn these new ones back to using .nvmrc